packages/components/src/components/contract-card/contract-card-items/turbos-card-body.tsx

```diff
@@ -8,6 +7,45 @@ import Icon from '../../icon';
 import MobileWrapper from '../../mobile-wrapper';
 import Money from '../../money';
 import { ResultStatusIcon } from '../result-overlay/result-overlay.jsx';
+import { TContractInfo, TGetContractUpdateConfig } from '@deriv/shared/src/utils/contract/contract-types';
+
+type TToastConfig = {
+    key?: string;
+    content: string;
+    timeout?: number;
+    is_bottom?: boolean;
+    type: string;
+};
+
+type TCardLabels = {
+    BARRIER_LEVEL: string;
+    BUY_PRICE: string;
+    CURRENT_PRICE: string;
+    STAKE: string;
+    TAKE_PROFIT: string;
+    TOTAL_PROFIT_LOSS: string;
+    PAYOUT: string;
+    PROFIT_LOSS: string;
+    POTENTIAL_PROFIT_LOSS: string;
+};
+type TTurbosCardBody = {
+    addToast: (toast_config: TToastConfig) => void;
+    connectWithContractUpdate?: any;
+    contract_info: TContractInfo;
+    contract_update?: TGetContractUpdateConfig['contract_update'];
+    currency: string;
+    current_focus?: string | null;
+    error_message_alignment?: string;
+    getCardLabels: () => TCardLabels;
+    getContractById: (contract_id: number) => any;
```

Comment:
`getContractById` function returns a very complicated object with lots of key-values pairs (and many of them are objects, arrays and functions). As this function hasn't been typed before it might take a lot of time to do it in this PR. `Any` is a temporary solution until the proper packaged won't been translated.

Reply:
```suggestion
    getContractById: (contract_id: number) => ContractStore;
```